### DISCUSSION TITLE
M1 mac and podman wich machine x86_64 arch

### DISCUSSION THREAD

**Discussion by replay111** [containers/podman#21528] [General]
Hi,

is it possible on M1 mac (or general on arm arch) create podman machine but with architecture x86_64?

I know that qemu can be runned in emulation mode - and I know it will be slow,  but I am just wondering if there is a such option just to run x86_64 arch machine on arm native hardware with podman?

**Comment by tnk4on** [comment]
Please see https://github.com/containers/podman/discussions/21539
